ambientweather2sqlite/server.py

```
@@ -135,10 +137,10 @@
 
 
 class Server:
-    def __init__(self, live_data_url: str, db_path: str, port: int, host: str):
+    def __init__(self, live_data_url: str, database: DatabaseManager, port: int, host: str):
```

Ruff (0.11.9)

Line too long (92 > 88)

(E501)